integration/swarm/swarm.bats

```diff
@@ -8,6 +8,8 @@
 # Swarm testing : This will start swarm as well as it will create and
 # run swarm replicas using Nginx
 
+load "${BATS_TEST_DIRNAME}/../../metrics/lib/common.bash"
```

Comment:
Sorry, late to the party...
So, if we are going to start using these check functions outside of the metrics (and we should...), then really we should probably lift the code out of the metrics lib and into the .ci/lib.sh more global place. Which, iirc, the metrics then pulls in anyway, so hopefully that code move should be less pain that it may have otherwise been.